### PR TITLE
Update French Language

### PR DESCRIPTION
**Pull request by jamperezmondragon**
I translated the missing messages. I am not a native french speaker, but I asked native speakers to review my work. I ran local tests and everything seemed okay.

### CODE REVIEW

File: selfdrive/ui/translations/main_fr.ts [hunk 1]
```diff
@@ -264,11 +264,11 @@
     </message>
     <message>
         <source>Pair Device</source>
-        <translation type="unfinished"></translation>
+        <translation>Associez l&apos;appareil</translation>
     </message>
     <message>
         <source>PAIR</source>
-        <translation type="unfinished"></translation>
+        <translation>Assoiez</translation>
```
Comment (robbederks) [inline selfdrive/ui/translations/main_fr.ts:271]: Associez?

File: selfdrive/ui/translations/main_fr.ts [hunk 2]
```diff
@@ -412,23 +412,23 @@
     <name>OnroadAlerts</name>
     <message>
         <source>openpilot Unavailable</source>
-        <translation type="unfinished"></translation>
+        <translation>openpilot Indisponible</translation>
```
Comment (maxime-desroches) [inline selfdrive/ui/translations/main_fr.ts:415]:
```suggestion
        <translation>openpilot indisponible</translation>
```

File: selfdrive/ui/translations/main_fr.ts (hunk 1 above)
Comment (jamperezmondragon) [reply selfdrive/ui/translations/main_fr.ts:271]: First I thought "Coupler" but I saw that "Associer" is the verb used in all other instances ("coupler" never appears). I wanted the new translations to be consistent with the text. What do you think? should we change those other instances to "Coupler"? Or another verb? Also maybe we should put this one in all caps right... Let me change that.

**Comment by jamperezmondragon** [comment]
Hello, I've been trying to understand the error messages, how the tests work etc... I'm not exactly sure why the tests fail, can anyone give me a pointer on what to do? Also, should we convert this PR to a draft? Maybe other people still have feedback on the translations?

**Pull request closed without merging**